<ui version="4.0" >
 <class>History</class>
 <widget class="QDialog" name="History" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>View history</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resource.qrc" >
    <normaloff>:/message-history</normaloff>:/message-history</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <property name="spacing" >
      <number>15</number>
     </property>
     <property name="leftMargin" >
      <number>3</number>
     </property>
     <property name="topMargin" >
      <number>3</number>
     </property>
     <property name="rightMargin" >
      <number>3</number>
     </property>
     <property name="bottomMargin" >
      <number>9</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Contact:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="contactBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Filter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="filterBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text" >
           <string>All messages</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Only incoming</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Only outgoing</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible" >
      <bool>true</bool>
     </property>
     <widget class="QTreeWidget" name="dateTree" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="animated" >
       <bool>true</bool>
      </property>
      <column>
       <property name="text" >
        <string>Date</string>
       </property>
      </column>
     </widget>
     <widget class="QTextBrowser" name="messageBrowser" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>1</horstretch>
        <verstretch>5</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="searchWidget" >
     <layout class="QVBoxLayout" name="verticalLayout" >
      <property name="spacing" >
       <number>5</number>
      </property>
      <property name="margin" >
       <number>10</number>
      </property>
      <item>
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" >
        <item>
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Log contents:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="searchText" />
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="olderBox" >
            <property name="text" >
             <string>Only older than</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QDateEdit" name="olderDate" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QCheckBox" name="newerBox" >
            <property name="text" >
             <string>Only newer than</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QDateEdit" name="newerDate" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QPushButton" name="applyButton" >
            <property name="text" >
             <string>Apply filter...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>250</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <widget class="QPushButton" name="searchButton" >
       <property name="text" >
        <string>Search messages...</string>
       </property>
       <property name="icon" >
        <iconset resource="resource.qrc" >
         <normaloff>:/go-up</normaloff>:/go-up</iconset>
       </property>
       <property name="shortcut" >
        <string>Ctrl+F</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dateTree</tabstop>
  <tabstop>messageBrowser</tabstop>
  <tabstop>contactBox</tabstop>
  <tabstop>filterBox</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>History</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>660</x>
     <y>705</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>History</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>660</x>
     <y>705</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>History</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>660</x>
     <y>705</y>
    </hint>
    <hint type="destinationlabel" >
     <x>117</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>olderBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>olderDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>43</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel" >
     <x>169</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newerBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>newerDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>36</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel" >
     <x>182</x>
     <y>646</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
